app/scripts/modules/google/src/loadBalancer/configure/http/createHttpLoadBalancer.controller.js

```diff
@@ -15,7 +15,7 @@ module.exports = angular.module('spinnaker.deck.gce.loadBalancer.createHttp.cont
   require('./backendService/backendService.component.js'),
   require('./basicSettings/basicSettings.component.js'),
   GCE_CACHE_REFRESH,
-  require('core/modal/wizard/wizardSubFormValidation.service.js'),
+  require('@spinnaker/core/modal/wizard/wizardSubFormValidation.service.js'),
```

Comment:
You might need to convert this file to TS and export it in the global namespace (not now, necessarily, but before we split everything into its own repository).